inference/bigcode_base.py

```diff
@@ -3,14 +3,22 @@
 """
 import torch
 from transformers import AutoTokenizer, AutoModelForCausalLM
-from .local_huggingface_model import _stop_at_stop_token
-
+from local_huggingface_model import _stop_at_stop_token
```

Comment:
keep this as `.local_huggingface_model`

You can still run your fim_inference script from the root of the repository as `python3 -m inference.fim_inference`. You may have to rename the script to `fim_inference`, i.e., with an underscore.